CommandDotNet/OperandAttribute.cs

```diff
@@ -7,5 +7,6 @@ public class OperandAttribute : Attribute, INameAndDescription
     {
         public string Name { get; set; }
         public string Description { get; set; }
+        public bool AppendPipedInput { get; set; }
```

Comment:
To make PipedInput an optional middleware, we can't have it as part of a default attribute.  I think we either enable it for all list operands by default or create an AllowPipedInputAttribute.  The first is the simpler option for this exercise.  It can be enhanced later.   I'll update this branch to ensure this is an optional feature.